lib/jefferies_tube/railtie.rb

```diff
@@ -4,10 +4,13 @@
 require 'rails'
 
 module JefferiesTube
-
   class Railtie < ::Rails::Railtie
     railtie_name :jefferies_tube
 
+    # generators do
```

Comment:
Why is this commented out? Is it not needed?

Reply:
no, good catch.  it works through magic once i got the folders / class names correct but was a classically difficult to figure out rails magic thing